app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,92 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* typography */  
+  h1 {
+    font-size: 3em;
+    margin-bottom: 1em;
+    text-align: center;
+    color: #544a47;
+  }
+
+  h2 {
+    font-size: 2em;
+    margin-bottom: 1em;
+    text-align: center;
+    color: #544a47;   
+  }
+  
+  p {
+    font-size: 1.1em;
+    line-height: 1.7em;
+    color: #544a47;
+  }
+
+  a {
+    color: #544a47;
+    &:hover {
```

こういう境目は空行挟むのが一般的ですね 🙆 
```suggestion
    color: #544a47;

    &:hover {
```